lib/Models/WebMapServiceCatalogItem.ts

```diff
@@ -954,6 +954,7 @@ class WebMapServiceCatalogItem
       let rectangle;
 
       if (
+        this.clipToRectangle &&
         this.rectangle !== undefined &&
         this.rectangle.east !== undefined &&
```

Comment:
Should we perhaps remove these individual checks for each east, west, north, south so that we have a uniform behavior for both WMS & ArcGIS? Their values [default to 0](https://cesium.com/docs/cesiumjs-ref-doc/Rectangle.html?classFilter=rectangle).

Reply:
Huh we'll there you go, I should've checked the defaults, definitely need to work out how to consider them because that could have unintended consequences!

Reply:
Actually @na9da I think they default to undefined, this is our rectangle, not the Cesium Rectangle
https://github.com/TerriaJS/terriajs/blob/next/lib/Traits/MappableTraits.ts#L7-L35

Reply:
I meant in [this call](https://github.com/TerriaJS/terriajs/pull/4827/files/0f27ceaba973793f13e79f65a4255d56a689d64d#diff-6ac01432fad81fa96581f43b19dfd0f3L555) if one of east, west, south, north is undefined it will default to 0. So right now if the catalog definition is like `{west: 30, east: 50, south: -10}` then for WMS no imagery will be shown (because north is undefined), but for ArcGIS we show imagery for the region  `{west: 30, east: 50, north: 0, south: -10}`.

It is a trivial thing but good to be consistent, right?

Reply:
Ahh ok, it's hard to imagine a situation where you might end up with a partial rectangle but anything could happen I guess!